src/style.css

```
@@ -1 +1,43 @@
-/* styles go here */
+label {
+  display: block;
+}
+
+.label {
+  margin-bottom: 10px;
+}
+
+.accept {
+  margin-bottom: 0;
+}
+
+.password {
+  margin-bottom: 0;
+}
+
+.rate {
```

напиши через запятую
```suggestion
.accept,
.password,
.rate {
```